VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -64,24 +73,24 @@ final internal class ResponseCache
     {
         let key = request.cacheKey
         
-        self.memoryCache.removeResponseDictionaryForKey(key)
```

Comment:
Consider keeping the method names more simple / generic. I.e. dropping the term "response dictionary" and just using "response"